cmd/common.go

```diff
@@ -184,14 +186,14 @@ func prettyPrint(r *check.Controls, summary check.Summary) {
 
 	// Print remediations.
 	if !noRemediations {
-		if summary.Fail > 0 || summary.Warn > 0 {
+		if summary.Fail > 0 || summary.Warn > 0 || summary.Manu > 0 || summary.Erro > 0 {
 			colors[check.WARN].Printf("== Remediations %s ==\n", r.Type)
 			for _, g := range r.Groups {
 				for _, c := range g.Checks {
-					if c.State == check.FAIL {
+					if c.State == check.FAIL || c.State == check.ERRO {
 						fmt.Printf("%s %s\n", c.ID, c.Remediation)
 					}
-					if c.State == check.WARN {
+					if c.State == check.WARN || c.State == check.MANU {
 						// Print the error if test failed due to problem with the audit command
 						if c.Reason != "" && c.Type != "manual" {
```

Comment:
WARN should be with FAIL and here it should be ERRO since Reason was a warn_reason which is why a test returned error, now WARN is just a fail test with no score. and ERRO is the test with reasons like command not exist or invalid parameter. 
I think maybe we should consider change 'warn_reason' in the json output to error_reason or something like this.

Reply:
I'm not sure I understood what `warn_reason` is, since there is only `reason` in json output.